### DISCUSSION TITLE
Replacing workspace icons?

### DISCUSSION THREAD

**Discussion by CG573** [i3/i3#6277] [Q&A] [answered]
I am running EndeavourOS with i3wm. For workspaces in the panel bar worspaces 1-5 have icons rather than numbers, is there a way to replace these icons? Thanks in advance!

**Comment by unt3rhofer** [comment, marked as answer]
Yup, you should be able to go to 
~/.config/i3/config and edit that file with your preffered text editor. 
There you will find a block of "code" that looks something like this (this is pasted from my config):
set $ws2 "2"
set $ws3 "3"
set $ws4 "4#1"
set $ws5 "5#2"
set $ws6 "63"
set $ws7 "74"
set $ws8 "8"
set $ws9 "9"
set $ws10 "10"
(it looks weird here because I use custom Font Awesome icons but I think you get the point.

There you can edit the name of your workspaces any way you like! (If you want fancy icons i recommend installing font awesome 4 and copy pasting icons from their cheatsheet: https://fontawesome.com/v4/cheatsheet/  )

Hope this helps!
